robots/accession/shelve.rb

```diff
@@ -9,11 +12,9 @@ def initialize
 
         def perform(druid)
           obj = Dor.find(druid)
-          obj.shelve if obj.respond_to? :shelve
+          Dor::Shelve.push(obj) if obj.is_a? Dor::Item
```

Comment:
Can you say more about this change? It's not clear as a reviewer that they are functionally equivalent.

Reply:
The `shelve` method is deprecated on dor-services because it violates the SRP. https://github.com/sul-dlss/dor-services/pull/393

Reply:
Dor::Item is the only object that includes `Shelvable`: https://github.com/sul-dlss/dor-services/blob/523b84df36376cc2a50cc08b45e26616ec8cc670/lib/dor/models/item.rb#L5